ckanext/harvest/harvesters/ckanharvester.py

```diff
@@ -21,22 +22,24 @@ class CKANHarvester(HarvesterBase):
     '''
     config = None
 
-    api_version = 2
+    api_version = 3
     action_api_version = 3
 
-    def _get_rest_api_offset(self):
-        return '/api/%d/rest' % self.api_version
-
     def _get_action_api_offset(self):
         return '/api/%d/action' % self.action_api_version
 
     def _get_search_api_offset(self):
-        return '/api/%d/search' % self.api_version
+        return '/api/2/search'
 
+    def _get_rest_api_offset(self):
+        return '/api/2/rest'
+    
     def _get_content(self, url):
         http_request = urllib2.Request(
             url = url,
         )
+	
+	http_request.add_header("User-Agent", "doi_harvest")
```

Comment:
@jbrown-xentity shouldn't this be using the harvest package name?

Reply:
This should have always been data_gov_harvest, not DOI. This isn't always relevant, but helps get around some web blockers who block traffic for empty user agents. I've seen this on another project, and thought it would be useful to implement here.